Makefile

```diff
@@ -74,6 +74,23 @@ GOVERALLS = $(GOBIN)/goveralls
 $(GOBIN)/goveralls: | $(BASE) ; $(info  building goveralls...)
 	$Q env GO111MODULE=off go get github.com/mattn/goveralls
 
+MOCKERY = $(shell pwd)/bin/mockery
+mockery: ## Download mockery if necessary.
+	$(call go-get-tool,$(MOCKERY),github.com/vektra/mockery/v2@v2.8.0)
```

Comment:
go-get-tool helper will not work for go1.18. I propose to use go install instead
```
MOCKERY = $(shell pwd)/bin/mockery
mockery: ## Download mockery if necessary.
        $(GO) install github.com/vektra/mockery/v2@v2.8.0
```